### ISSUE TITLE
Open Search for TMT16/TMT18-plex with Extended Precursor Mass Range (lower than -230 Da, higher than 600 Da)

### ISSUE THREAD

**Issue by jmrodriguezc** [Nesvilab/MSFragger#340]
Hello everyone,  

We are trying to perform an open search for TMT18-plex, but we have encountered an issue: the lower bound of the precursor mass window is limited to -230 Da.  

Is there any way to perform an open search with a precursor mass range of -350 Da to 600 Da?  

Thank you in advance for your assistance.  

Best regards,  
J

**Comment by jmrodriguezc** [comment]
Thank you for your quick response, Fengchao and Alexey Nesvizhskii.

The issue lies within the search engine, as we are running MSFragger via the command line. After this, we use post-processing programs to model the modifications. Therefore, we need to extend the range of precursor masses directly in MSFragger.

Best regards,
J

**Comment by fcyu** [comment]
Yes, we hardcoded it in MSFragger. I will remove it in the next release.

Best,

Fengchao

**Comment by jmrodriguezc** [comment]
Thank you for your consideration, Fengchao and Alexey Nesvizhskii.

Best Regards,
J

**Comment by jmrodriguezc** [comment]
Hi Fengchao,

BTW, do you know when the release with this update will be available?

Thanks in adavance!

Best,
J
